recipes/starshot_card_block_type/config/field.field.block_content.card.body.yml

```diff
@@ -0,0 +1,23 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - block_content.type.card
+    - field.storage.block_content.body
+  module:
+    - text
+id: block_content.card.body
+field_name: body
```

Comment:
I think if you're going to use `field_` elsewhere then use it for everything, as other than being persistent "body" is just a field and shouldn't have special treatment.

Reply:
Except that we're using the `body` field that ships with the `block_content` module, so it's kind of an inherited decision.

I don't feel super strongly but I'd rather not essentially create new fields where we can reuse them.

Reply:
You can just delete the body field, it doesn't break anything. We shouldn't be limited by old core decisions of which this one isn't great.

Note that i advocate no `field_` prefixing, which would make `body` good. https://www.youtube.com/watch?v=2495KafRZAg

Reply:
I don't think we really need to get too hung up on the fields' internal names.

Starshot has no update path, so we can always change this later, whenever we want. I think it's safe to assume that end users probably won't care about the names; since Starshot is meant to put users first, I think it makes sense to merge this if the data model overall makes sense, and we can bikeshed the field names in a separate PR.

Reply:
We have to be mindful that decisions made 10 years set what people think is best practice. So we have to start a dialog about what we're pushing out (and have a way to defer gold plating without dropping the ball). I really would love to see a target content model done in Excalidraw or something so that we can measure PRs against it.

Reply:
I'm in favor of developing some reasonable guidelines around field names, but this repository is a rapid prototype of Starshot developed by mostly by fiat, with the goal of delivering as much value to end users as we can, as quickly as we can. So we can play a little fast and loose, at least for now. :)